spiral/framework
# Added RoadRunner 2.0 support

| Q             | A
| ------------- | ---
| Bugfix?       | ❌
| Breaks BC?    | ✔️/❌ (Depending on the RoadRunner version)
| New feature?  | ✔️

Added RoadRunner 2.0 support.

## Manual Testing

Since not all libraries currently support `spiral/roadrunner` 2.0, testing can be done in the following way.

1) In the existing composer file, you will need to replace the dependency with the following:
```json
{
    "require": {
        "spiral/roadrunner": "2.0.0-RC.4 as 1.9.2"
    }
}
```

2) You should remove the `./spiral` (or `./spiral.exe`) binary and replace it with `./rr` (or `./rr.exe`) using following command:
```sh
$ ./vendor/bin/rr get --stability=RC
```

3) Don't forget to update the `.rr.yaml` configuration
